src/components/Cascader.astro

```diff
@@ -0,0 +1,53 @@
+---
+import { getHeadings } from "../assets/mock.md";
```

Comment:
can you change this so that this component takes `MarkdownHeading[]` as a prop directly? that will make it easier to reuse this component on different pages.